### PR TITLE
LPS-183276 | Create a macro to delete all datasetViews via API

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
Create a macro to delete all datasetViews via API.

**JIRA Link**
https://issues.liferay.com/browse/LPS-183276

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
just forced this PR to resolve rebase conflicts

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
I'm on it 🕵🏽

**Comment by mateusralv** [comment]
ci:test:frontend-data-set

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:frontend-data-set

**Comment by mateusralv** [comment]
hi @abhnerramos could you take a look and tell me what you think? thanks :)

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro [hunk 1]
```diff
@@ -130,6 +132,17 @@ definition {
 			value1 = "No Results Found");
 	}
 
+	macro deleteAllDatasetViews {
```
Comment (abhnerramos) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro:135]: shouldn't this be deleteAllDatasetViews**ViaAPI**?

**Comment by mateusralv** [comment]
ci:forward

**Comment by abhnerramos** [comment]
ci:forward

**Pull request closed without merging**